plugins/mumble_plugin_linux.h

```diff
@@ -291,29 +255,30 @@ static inline procptr_t getExportedSymbol(const std::string &symbol, const procp
 	// uint32_t chain[nChain];
 	const auto nChain = peekProc< uint32_t >(hashTable + sizeof(uint32_t));
 
-	if (is64Bit) {
-		for (uint32_t i = 0; i < nChain; ++i) {
-			const auto sym  = peekProc< Elf64_Sym >(symTable + sizeof(Elf64_Sym) * i);
-			const auto name = peekProcString(strTable + sym.st_name, symbol.size());
-
-			if (name == symbol) {
-				return module + sym.st_value;
-			}
-		}
-	} else {
-		for (uint32_t i = 0; i < nChain; ++i) {
-			const auto sym  = peekProc< Elf32_Sym >(symTable + sizeof(Elf32_Sym) * i);
-			const auto name = peekProcString(strTable + sym.st_name, symbol.size());
+	for (uint32_t i = 0; i < nChain; ++i) {
+		const auto sym  = peekProc< Elf_Sym >(symTable + sizeof(Elf_Sym) * i);
+		const auto name = peekProcString(strTable + sym.st_name, symbol.size());
 
-			if (name == symbol) {
-				return module + sym.st_value;
-			}
+		if (name == symbol) {
+			return module + sym.st_value;
 		}
 	}
 
 	return 0;
 }
 
+static inline procptr_t getExportedSymbol(const std::string &symbol, const procptr_t module) {
```

Comment:
The function name is the same as the template-function's one, right? I'm not sure that I like this. I would probably prefer these functions to have different names as their signature is the same as well (except for one being a template of course). This could get confusing :thinking:

Reply:
What about `getExportedSymbolInternal()`?

Consider that I'm eventually going to write a proper template class.

Reply:
Sounds good, yeah :+1: